### PR TITLE
fix sonar issues

### PR DESCRIPTION
**Pull request by Gr33ndev**
Part of https://github.com/Gamify-IT/issues/issues/498

https://sonar.sqa-kube-cluster4.cloudns.nz/dashboard?id=crosswordpuzzle

**Please test is everything is still working as intendet**

### CODE REVIEW

File: src/components/InputField.vue [hunk 1]
```diff
@@ -45,6 +45,7 @@ watch(
 );
 
 async function keyDown(key: KeyboardEvent) {
+  //NOSONAR
```
Comment (madebyTimo) [inline src/components/InputField.vue:48]: I think line 63 to 76 of this function are useless and can be removed.

Maybe then it is also possible to not use `NOSONAR`

**Review by madebyTimo — CHANGES_REQUESTED** [review]
good work except for the one function

File: src/views/CrosswordpuzzleView.vue [hunk 2]
```diff
@@ -102,6 +102,7 @@ function getWrongQuestionVertical(element: TileCrossWord): questionAnswer {
 }
 
 function evaluateSolution() {
+  //NOSONAR
```
Comment (delvh) [inline src/views/CrosswordpuzzleView.vue:105]: why exactly?

Comment (delvh) [reply src/views/CrosswordpuzzleView.vue:105]: Because this function is too large?
The better alternative would be to extract subfunctions.

**Comment by Gr33ndev** [comment]
Closed, because in the next sprint there will be big all-encompassing changes and it will be easier to fix the then new sonar issues.

**Comment by Gr33ndev** [comment]
Wrong PR

**Review by delvh — APPROVED** [review]
Should work.
The cyclomatic complexity problems were not tackled, but we can leave that for another PR.

**Pull request merged**